website/static/js/fangorn.js

```diff
@@ -1009,6 +1020,34 @@ function _removeEvent (event, items, col) {
     }
 }
 
+function doCheckout(item, checker, error_message) {
+    $.ajax({
```

Comment:
Functions that do asynchronous work should return promises. this allows for chaining.

```js
doCheckout(..)
    .then(function() {...})
```

In this case you can just return the jqXHR returned by `$.ajax`.